tools/releaseBuild/azureDevOps/templates/nuget.yml

```diff
@@ -136,6 +136,7 @@ jobs:
       outPathRoot: '$(System.ArtifactsDirectory)\signed'
       binVersion: $(SigingVersion)
       binVersionOverride: $(SigningVersionOverride)
+    condition: and(succeeded(), eq(variables['SHOULD_SIGN'], 'true'))
```

Comment:
Just curious, where is this variable defined?

Reply:
It get defined by calling the template in line 60